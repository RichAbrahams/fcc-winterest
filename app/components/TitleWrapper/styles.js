import styled from 'styled-components';

export const Wrapper = styled.div `
  display: flex;
  align-items: center;
  justify-content: center;
  width: 100%;
  position: relative;
  margin: 0rem 0rem 2rem 0rem;
  @media screen and (max-width: 850px) {
    flex-direction: column;
  }
`;

