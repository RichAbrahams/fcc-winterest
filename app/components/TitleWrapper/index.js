/**
*
* TitleWrapper
*
*/

import React from 'react';
import { Wrapper } from './styles';


function TitleWrapper(props) {
  return (
    <Wrapper className="title-wrapper">
      {props.children}
    </Wrapper>
  );
}

TitleWrapper.propTypes = {
  children: React.PropTypes.array,
};

export default TitleWrapper;
