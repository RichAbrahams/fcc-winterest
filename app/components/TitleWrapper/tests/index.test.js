import React from 'react';
import renderer from 'react-test-renderer';
import TitleWrapper from '../index';

it('renders correctly', () => {
  const tree = renderer.create(
    <TitleWrapper />).toJSON();
  expect(tree).toMatchSnapshot();
});
