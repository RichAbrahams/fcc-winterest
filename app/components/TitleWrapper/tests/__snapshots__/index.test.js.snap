exports[`test renders correctly 1`] = `
<div
  className="title-wrapper guBfrZ" />
`;
